packages/osd-opensearch/src/artifact.test.js

```diff
@@ -40,26 +40,34 @@ import { Artifact } from './artifact';
 const log = new ToolingLog();
 let MOCKS;
 
+const DAILY_SNAPSHOT_BASE_URL = 'https://artifacts.opensearch.org/snapshots/core/opensearch';
+// eslint-disable-next-line no-unused-vars
+const PERMANENT_SNAPSHOT_BASE_URL = '';
```

Comment:
Why are we keeping this variable if it isn't used?

Reply:
Part of the todo's to replace when we have a replacement if we have a replacement but I can remove it and when someone updates the file in the future they can update the tests to include testing it.